pkg/source/markdown.go

```diff
@@ -108,12 +114,22 @@ func (m *markdown) parse(filename string, opts Opts) (err error) {
 			count++
 		}
 
-		if count == 1 && !doxIdFound {
-			found, err := fmt.Sscanf(line, m.escape(doxIdFmt), &m.id)
+		if count == 1 || count == 2 {
```

Comment:
Can you help me understand this change?

Reply:
See https://github.com/jesselang/dox/pull/4#discussion_r241111472